src/pages/CreateCirclePage/CreateSampleCircle.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+
+import { useApeSnackbar, useApiBase } from '../../hooks';
+import { client } from '../../lib/gql/client';
+import { Button } from '../../ui';
+import { LoadingModal } from 'components';
+
+export const CreateSampleCircle = ({
+  onFinish,
+}: {
+  onFinish(circleId: number): void;
+}) => {
+  const { showError } = useApeSnackbar();
+  const [loading, setLoading] = useState(false);
+  const { fetchManifest } = useApiBase();
+
+  const createSampleCircle = async () => {
+    try {
+      setLoading(true);
+      const { createSampleCircle } = await client.mutate({
```

Comment:
a little weird to have this be the same as the function name
```suggestion
      const { createSampleCircle: created } = await client.mutate({
```

Reply:
changed